client/src/components/Post.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Post = props => {
+    return (
+        <li>
+            <h1>{props.post.title}</h1>
+            <p>{props.post.contents}</p>
+            <p>{props.post.created_at}</p>
```

Comment:
Destructuring would make this cleaner, either just `post` or even its properties as well:
```javascript
const Post = ({ post: { title, contents, created_at } }) => {
```
and, yes, nested destructuring is a thing, but I prefer this as it's cleaner and it does maintain access of `post` in case we need it:
```javascript
const Post = ({ post }) => {
  const { title, contents, created_at } = post;
```